### DISCUSSION TITLE
CPU Temperature going up and down.. is that normal?

### DISCUSSION THREAD

**Discussion by spanzetta** [jomjol/AI-on-the-edge-device#2836] [General] [answered]
Is that because of the taking pictures?
Can that explain the CPU temp going from ~53° to ~76° all the times?
All normal?

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/89143b24-f381-4868-a548-bf62230cb0df)

Thanks

**Comment by spanzetta** [comment]
Is there anyone that can comment if this is normal or not?
Thanks

**Reply by Taiger78** [reply]
Hello,
Mine is rather steady in 35 -37C I have never seen over 40C
I have it set to take pictures once / minute.

**Reply by Taiger78** [reply]
On the other hand I have not trended the temperature but only looked at the value from the web interface.(This as I have not been able to get MQTT working jet)

**Comment by spanzetta** [comment]
hmm... so my temperatur is NOT normal.. (on web interface I always read something about 75/77°)
Which can be the cause then?

**Reply by Slider0007** [reply, marked as answer]
Temp variation over the cycle is normal, because of variations in load. In idle temp is lower than during processing. So do not worry...

Be aware: Temp sensor in ESP32 is not officially supported, therefore it can be imprecise or unreliable. This can differ from device to device.

I have two devices a which are running the same software at avg. 75°C and the other at avg. 37°C next to each other...

**Reply by spanzetta** [reply]
I agree that in my case is a false value.. I measured the Chip temp with infrared temperature meter and it is less then 30°

Chip info from Info tab:
Chip Cores:	2
Chip Revision:	101
Chip Features:	50
